src/ResourceManager/Network/AzureRM.Network.psd1

```diff
@@ -291,7 +291,10 @@ CmdletsToExport = 'Add-AzureRmApplicationGatewayAuthenticationCertificate',
                'Get-AzureRmVirtualNetworkGatewayBgpPeerStatus', 
                'Get-AzureRmVirtualNetworkGatewayAdvertisedRoute', 
                'Get-AzureRmVirtualNetworkGatewayLearnedRoute', 
-               'Get-AzureRmNetworkUsage'
+               'Get-AzureRmNetworkUsage',
+               'Get-AzureRmVirtualMachineScaleSetPublicIPAddress',
```

Comment:
we should not add new cmdlets. Please look at Get-AzureRmNetworkInterface. this may not be auto generatable